doc/nrf/config_and_build/bootloaders_dfu/mcuboot_nsib/qspi_xip_split_image.rst

```diff
@@ -0,0 +1,295 @@
+.. _qspi_xip_split_image:
+
+Using QSPI XIP split image
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The QSPI XIP split image feature lets you gain more flash storage space for applications by splitting application code into two parts:
+
+* Code that runs on the internal flash memory.
+* Code that runs on supported external flash memory over the Quad Serial Peripheral Interface (QSPI) using Execute in Place (XIP).
+
+This feature is supported on nRF52840, nRF5340, and the SoCs of the nRF91 Series.
+
+.. warning::
+    On the nRF52840, do not relocate interrupts to the QSPI XIP flash.
+    Doing so can lock up or brick the device by making the debug access port inaccessible.
+
+The QSPI XIP split images are supported in :doc:`MCUboot <mcuboot:index-ncs>`, which allows for updating them over-the-air.
+
+For the nRF5340 DK and Nordic Thingy:53, you can also check out the :ref:`SMP Server with external XIP <smp_svr_ext_xip>` sample, which demonstrates this feature.
+
+Requirements
+************
+
+Make sure you meet the following requirements to make use of this feature:
+
+* Board based on nRF52840, nRF5340, or nRF91 Series SoCs, with file structure compatible with :ref:`Hardware model v2 (HWMv2) <hwmv1_to_v2_migration>`
+* External QSPI flash chip with supported commands connected to QSPI pins
+* QSPI flash chip in always-on mode (meaning, no DPM or low-power modes)
+* :doc:`MCUboot configuration <mcuboot:design>` in the Swap using move mode (``MCUBOOT_SWAP_USING_MOVE``), the Upgrade only mode (``MCUBOOT_OVERWRITE_ONLY``), or in the direct-XIP mode
+* :ref:`Static partition manager file <ug_pm_static>`
+* Linker file with QSPI XIP flash offset and size
+* :ref:`configuration_system_overview_sysbuild`
+
+.. caution::
+    At present, MCUboot images cannot be linked together.
```

Comment:
internal and external images are statically linked together.
problem is that linking can't be done dynamically in runtime.

Reply:
Comment relates to linking of dependencies e.g. you cannot have an update where the images are linked together, you can wrongly update one and not the other which will brick the unit

Reply:
I knew the limitations. My comments i about 'MCUboot images cannot be linked together.' meaning.
These image are actually linked together tough (by the linker). MCUboot doesn't support any method for interfacing dependent images (and it won't).